test/pamela/HTN/symbol-ref-htn.main.htn.edn

```diff
@@ -0,0 +1,372 @@
+ :hedge-63
+ {:display-name "1", :end-node :hem-52, :type :edge, :uid :hedge-63},
+ :hedge-65 {:end-node :hpt-39, :type :edge, :uid :hedge-65},
+ :hedge-66 {:end-node :hpt-40, :type :edge, :uid :hedge-66},
+ :hedge-67 {:end-node :hpt-41, :type :edge, :uid :hedge-67},
+ :hedge-68 {:end-node :hpt-42, :type :edge, :uid :hedge-68},
+ :hedge-69 {:end-node :hpt-43, :type :edge, :uid :hedge-69},
+ :hedge-70 {:end-node :hpt-44, :type :edge, :uid :hedge-70},
+ :hedge-71 {:end-node :hpt-45, :type :edge, :uid :hedge-71},
+ :hedge-72 {:end-node :hpt-46, :type :edge, :uid :hedge-72},
+ :hedge-73 {:end-node :hpt-47, :type :edge, :uid :hedge-73},
+ :hedge-74 {:end-node :hpt-48, :type :edge, :uid :hedge-74},
+ :hedge-75 {:end-node :henpt-49, :type :edge, :uid :hedge-75},
+ :hedge-76
+ {:display-name "12", :end-node :hem-55, :type :edge, :uid :hedge-76},
+ :hedge-78 {:end-node :henpt-50, :type :edge, :uid :hedge-78},
+ :hedge-79
+ {:display-name "13", :end-node :hem-57, :type :edge, :uid :hedge-79},
+ :hedge-81 {:end-node :henpt-51, :type :edge, :uid :hedge-81},
+ :hedge-82
+ {:display-name "14", :end-node :hem-59, :type :edge, :uid :hedge-82},
+ :hem-36
+ {:args [],
+  :display-args [],
+  :display-name "Main",
+  :edges [:hedge-63],
+  :incidence-set #{:hedge-61},
+  :network :net-62,
+  :type :htn-expanded-method,
+  :uid :hem-36},
+ :hem-52
+ {:args [{:type :field-ref, :names [field1]} 42],
+  :display-args [field1 42],
+  :display-name "Reference Tests",
+  :edges [:hedge-76 :hedge-79 :hedge-82],
+  :incidence-set #{},
+  :network :net-64,
+  :type :htn-expanded-method,
+  :uid :hem-52},
+ :hem-55
+ {:args [4 5 :running],
+  :display-args [4 5 :running],
+  :display-name "Method 3",
+  :edges [],
+  :incidence-set #{},
+  :network :net-77,
+  :type :htn-expanded-method,
+  :uid :hem-55},
+ :hem-57
+ {:args [4 5 :stopped],
+  :display-args [4 5 :stopped],
+  :display-name "Method 3",
+  :edges [],
+  :incidence-set #{},
+  :network :net-80,
+  :type :htn-expanded-method,
+  :uid :hem-57},
+ :hem-59
+ {:args
+  [4
+   5
+   {:type :mode-ref,
+    :mode-ref {:type :symbol-ref, :names [locations]},
+    :mode :sea}],
+  :display-args [4 5 (mode-of locations :sea)],
+  :display-name "Method 3",
+  :edges [],
+  :incidence-set #{},
+  :network :net-83,
+  :type :htn-expanded-method,
+  :uid :hem-59},
+ :henpt-34
+ {:display-name "main",
+  :edges [:hedge-61],
+  :incidence-set #{},
+  :type :htn-expanded-nonprimitive-task,
+  :uid :henpt-34},
+ :henpt-35
+ {:args
+  [{:type :pclass-ctor,
+    :uid :pclass-2,
+    :pclass pclass1,
+    :args [42],
+    :fields
+    {field1-1 {:access :private, :observable false, :initial 42}},
+    :ancestry [[:pclass-1 field1]]}
+   42],
+  :argsmap
+  {"method-arg" 42,
+   "pclass-instance-arg"
+   {:ancestry [[:pclass-1 field1]],
+    :args [42],
+    :fields
+    {field1-1 {:access :private, :initial 42, :observable false}},
+    :pclass pclass1,
+    :type :pclass-ctor,
+    :uid :pclass-2}},
+  :display-args [(nil 42) 42],
```

Comment:
The 1st arg is a field reference, so it should be simply:
`:display-args [field1 42]`